index.js

```diff
@@ -459,6 +452,14 @@ BloomClient.prototype._onConnect = function () {
 
   this.unavailable = false
 
+  this.responseParser = new ResponseParser(this)
```

Comment:
nit: do we need a new ResponseParser for each connection? i.e. could we still do this.responseParser initialisation in the constructor?